src/AngleSharp.Css/Values/Composites/CssOriginValue.cs

```diff
@@ -73,14 +74,17 @@ public String CssText
         /// <returns>True if both are equal, otherwise false.</returns>
         public Boolean Equals(CssOriginValue other)
         {
-            return _x.Equals(other._x) && _y.Equals(other._y) && _z.Equals(other._z);
+            return other is not null
+                && EqualityComparer<ICssValue>.Default.Equals(_x, other._x)
```

Comment:
This would be different to the other values. Any reason for changing it (I think it was mainly / exclusively taken to avoid checking for the possible `null` states)? If you want to change it you'd need to change it in all values to be consistent.

Personally, I'm fine with having this as an intermediate step, but we'd definitely need to make sure that this is consistent at the end of the day (i.e., when fully releasing 1.0.0). To make it a bit more efficient / less repetitive I'd write it like

```cs
if (other is not null)
{
  var cmp = EqualityComparer<ICssValue>.Default;
  return cmp.Equals(_x, other._x) && cmp.Equals(_y, other._y) && cmp.Equals(_z, other._z);
}

return false;
```

(Just to emphasize: I am fine with the change, it should just be consistent)

Reply:
As you like - as mentioned we can use this as a stepping stone (i.e., make it in 2 PRs, where the *who* is left undecided - if you want to take it I'd appreciate it though).

Reply:
I would appreciate it if you could publish a new preview version quickly. So, I would prefer doing another PR later (this evening or maybe later this week) to update other files.

Reply:
New PR: https://github.com/AngleSharp/AngleSharp.Css/pull/163